lib/chainer/functions/array/reshape.rb

```diff
@@ -0,0 +1,26 @@
+module Chainer
+  module Functions
+    module Array
+      # Reshapes an input array without copy.
+      class Reshape < Function
+        def initialize(shape)
+            @shape = shape
+        end
+
+        def self.reshape(xs, shape)
+          self.new(shape).(xs)
+        end
+
+        def forward_cpu(xs)
```

Comment:
Here, you define the forward_cpu(), backward_cpu() method.
If there is no plans to separate the code between the CPU and the GPU in the future, I think that the forward(), backward() names are better as in the original chainer/functions/array/reshape.py.